src/ripple/app/misc/impl/Manifest.cpp

```diff
@@ -309,6 +309,30 @@ ManifestCache::getMasterKey (PublicKey const& pk) const
     return pk;
 }
 
+std::uint32_t
+ManifestCache::getSequence (PublicKey const& pk) const
+{
+    std::lock_guard lock{read_mutex_};
+    auto const iter = map_.find (pk);
+
+    if (iter != map_.end () && !iter->second.revoked ())
```

Comment:
I'm not sure if returning `0` is appropriate for a revoked manifest.

Reply:
Changed this to boost::optional and boost::none as suggested below